### PR TITLE
feat:(GUI): Merge 2.x into main after merging whitebear60's i18n into 2.x

### PR DESCRIPTION
**Pull request by Griefed**

### CODE REVIEW

File: src/main/java/de/griefed/serverpackcreator/CLISetup.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+    /** A helper method for config setup. Prompts user to enter boolean values that will be stored in config and checks entered values to prevent storing non-boolean values in boolean variables.
+    * @return Boolean. Converts to boolean and returns value entered by user that will be stored in config.
+    */
+    private boolean readBoolean() {
+        Scanner readerBoolean = new Scanner(System.in);
+        String boolRead;
+        while (true) {
+            boolRead = readerBoolean.nextLine();
+            if (boolRead.matches("1")                                                           ||
```
Comment (whitebear60) [inline src/main/java/de/griefed/serverpackcreator/CLISetup.java:218]: Can I request to add back way to input [Yy]es or [Nn]o in English keyboard layout regardless of the current language chosen as I have designed it initially today in the morning?

**Comment by whitebear60** [comment]
GitHub think that I have started the review... I needed to leave that comment here...

**Review by whitebear60 — CHANGES_REQUESTED** [review]

File: src/main/java/de/griefed/serverpackcreator/CLISetup.java (hunk 1 above)
Comment (Griefed) [reply src/main/java/de/griefed/serverpackcreator/CLISetup.java:218]: Sure thing.

Comment (Griefed) [reply src/main/java/de/griefed/serverpackcreator/CLISetup.java]: See https://github.com/Griefed/ServerPackCreator/pull/45/commits/ca2219e48092372cfa7db8f50ccc20d87d2b4e5d

**Comment by Griefed** [comment]
Regarding ServerPackCreatorLog and ModloaderInstallerLog:
TODO: Find more efficient way of displaying logs in UI. The currently implemented, commented out, way bloats memory usage by ungodly amounts(from ~180MB-30MB to ~2GB, wtf).

**Pull request merged**